webapp/channels/src/components/with_tooltip/index.tsx

```diff
@@ -17,6 +17,8 @@ type WithTooltipProps = {
     children: OverlayTriggerProps['children'];
     placement: OverlayTriggerProps['placement'];
     onShow?: () => void;
+    delayHide?: number;
```

Comment:
I wonder if the delay hide is needed really in this case. Do you know what is the impact of removing this delay?

Reply:
I don't know why it was added here, but there is a delay of 600ms if the tooltip message contains URL.
https://github.com/mattermost/mattermost/blob/96be8d5a584eadfc9f291064617d061dc494e086/webapp/channels/src/components/announcement_bar/default_announcement_bar/announcement_bar.tsx#L75

Reply:
I went digging, and found the reason for this delay: https://github.com/mattermost/mattermost-webapp/pull/5907#issuecomment-680290989

Do you mind checking if this reasoning still applies? (i.e. without the delay it becomes really hard to interact with the url in the tooltip)

Reply:
I've checked it and when I removed delay it was impossible to click the URL in tooltip, so I think that the delay is still needed here.